lab1/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="app.css">
+    <meta charset="UTF-8">
+    <title>Sushi Fantasy</title>
+</head>
+<body>
+    <h1>Sushi Fantasy</h1>
+
+    <div id=mainpara>
```

Comment:
It's better off to be safe than sorry. Surround the value of the attribute with quote. (e.g. id="mainpara")